tmeister/dataaccess/toggleda.py

```diff
@@ -1,12 +1,13 @@
 from asyncpgsa import pg
 from sqlalchemy.sql import functions
+from sqlalchemy import func
 
 from . import db
 
 
 async def get_toggle_states_for_env(env, list_of_features):
     query = db.toggles.select() \
-        .where(db.toggles.c.env == env) \
+        .where(func.lower(db.toggles.c.env) == env) \
```

Comment:
Should `env` also be lowercased?

Reply:
Wait, nm...I see https://github.com/CanopyTax/toggle-meister/pull/32/files#diff-1ad65ff9a82e48980759fe1d26e0c2afR40